spotbugs/src/main/java/edu/umd/cs/findbugs/io/IO.java

```diff
@@ -182,6 +183,22 @@ public static void close(@CheckForNull AutoCloseable c) {
         }
     }
 
+    /**
+     * Close given Closeable instance, ignoring any resulting exception.
+     *
+     */
+    public static void close(@CheckForNull Closeable c) {
```

Comment:
This could be written as `    public static void close(@CheckForNull Closeable c) {
        close((AutoCloseable) c);
    }`

Reply:
possible, I will add a commit to fix.